lib/modules/datasource/maven/util.ts

```diff
@@ -126,6 +127,20 @@ export async function downloadHttpProtocol(
       if (isTemporaryError(err)) {
         logger.debug({ failedUrl, err }, 'Temporary error');
         if (getHost(url) === getHost(MAVEN_REPO)) {
+          const statusCode = err?.response?.statusCode;
+          if (statusCode === 429) {
+            if (getCacheType() === 'redis') {
+              logger.warn(
```

Comment:
are we sure we should warn here?

Reply:
Yes, it's important to Renovate users to know if they are exceeding Maven Central rate limits as they risk further blocking. Hopefully this scenario (with caching) doesn't happen, but if it does - they should know